src/WorkingSetSelectorComponent.cpp

```diff
@@ -91,6 +90,11 @@ void WorkingSetSelectorComponent::updateIopLoadIndicators()
 	}
 }
 
+int WorkingSetSelectorComponent::buttonPadding()
+{
+	return (WIDTH - BUTTON_WIDTH) / 2.0;
```

Comment:
Nit - did you use a `double` on purpose here?

I'd either do this
```suggestion
	return ((WIDTH - BUTTON_WIDTH) / 2);
```
Or cast it appropriately I guess